src/DiscreteBayesNet/structure_scoring.jl

```diff
@@ -0,0 +1,18 @@
+"""
+    bayesian_score(g::DAG, names::Vector{Symbol}, data::DataFrame, prior::DirichletPrior=UniformPrior())
+
+Compute the bayesian score for graph structure `g`, with the data in `data`. `names` containes a symbol corresponding to each vertex in `g` that is the name of a column in `data`.
+
+Note that every entry in data must be an integer greater than 0
```

Comment:
In principle, with Julia's type system it should be possible to enforce this without a lot of degradation in performance. (But then I don't know how optimal Dataframe would be for the task.)
But anyway it was just a suggestion. Not a big issue :smiley: